cli/main.go

```diff
@@ -28,6 +37,98 @@ func main() {
 		EnableBashCompletion:   true,
 		UseShortOptionHandling: true,
 		Commands: []*cli.Command{
+			{
+				Name:  "status",
+				Usage: "Checks the status of your eigenpod.",
+				Flags: []cli.Flag{
+					&cli.BoolFlag{
+						Name:        "json",
+						Value:       false,
+						Usage:       "print only plain JSON",
+						Required:    false,
+						Destination: &useJson,
+					},
+				},
+				Action: func(cctx *cli.Context) error {
+					eth, err := ethclient.Dial(node)
+					PanicOnError("failed to reach eth --node.", err)
+
+					beaconClient, err := getBeaconClient(beacon)
+					PanicOnError("failed to reach beacon chain.", err)
+
+					status := getStatus(ctx, eigenpodAddress, eth, beaconClient)
+					eigenpod, err := onchain.NewEigenPod(common.HexToAddress(eigenpodAddress), eth)
+					PanicOnError("failed to reach eigenpod", err)
+
+					if useJson {
+						bytes, err := json.MarshalIndent(status, "", "      ")
+						PanicOnError("failed to get status", err)
+						statusStr := string(bytes)
+						fmt.Println(statusStr)
+					} else {
+						// pretty print everything
+						color.New(color.Bold, color.FgBlack).Printf("Eigenpod validators\n")
```

Comment:
I can't see this on my computer

Reply:
Does your terminal not support ansi color codes xD

Reply:
it definitely shows up as black but my terminal background is like... plum colored? it's the default for ubuntu. so this text is nearly invisible.